Gopkg.toml

```diff
@@ -36,6 +37,10 @@
   name = "github.com/cheggaaa/pb"
   version = "1.0.28"
 
+[[constraint]]
+  name = "github.com/google/uuid"
+  version = "v1.1.1"
```

Comment:
Can you just stick with the major version instead of this specific version?
I mean, we want to use the latest v1 version